collectors/APICallCollector.js

```diff
@@ -48,7 +48,7 @@ class APICallCollector extends BaseCollector {
      */
     async onExecutionContextCrated(trackerTracker, params) {
         // ignore context created by puppeteer / our crawler
-        if (!params.context.origin && params.context.auxData.type === 'isolated') {
+        if ((!params.context.origin || params.context.origin === '://') && params.context.auxData.type === 'isolated') {
```

Comment:
context with `origin === '://'` are also pupeteers internal contexts, so we can skip them